main/dialog/APKBUILD

```diff
@@ -30,6 +30,9 @@ build() {
 package() {
 	cd "$builddir"
 	make DESTDIR="$pkgdir" install
+
+	# Get rid of static library
+	rm -rf "$pkgdir"/usr/lib
```

Comment:
Wouldn't it be better to split this into a `-static` subpackage instead?

Reply:
ok